packages/components/organisms/f-cookie-banner/test-utils/component-objects/f-cookieBanner.component.js

```diff
@@ -1,5 +1,20 @@
-const cookieBannerComponent = () => $('[data-test-id="cookieBanner-component"]');
+const Page = require('@justeat/f-wdio-utils/src/page.object');
 
-exports.waitForCookieBannerComponent = () => cookieBannerComponent().waitForExist();
+class CookieBanner extends Page {
```

Comment:
Very tiny suggestion - I've used `module.exports = class CookieBanner extends Page` in other components, to get rid of the extra line of code on line 20, could be worth doing that here?